src/delta_lake/data_store.py

```diff
@@ -200,3 +200,33 @@ def _get_structure(
         db_structure = _execute_with_spark(_get_structure, spark)
 
     return _format_output(db_structure, return_json)
+
+
+def is_database_exists(
```

Comment:
Unless you're going for a java thing I'd drop the `is_` for these two methods

Reply:
👍